contribs/gmf/examples/style/mobileapp-vars.less

```diff
@@ -1,8 +1,13 @@
+@app-margin: 13px;
```

Comment:
> Let's remove the geolocation button for now,  and re-add it later when functional.

That make sens. I'll remove the html "geolocation" button.

>I 'd really prefer the zoom and geolocation buttons to be displayed using the same classes.

I don't think that this is a good idea to link somehow two completely different functionalities :-/